docs/sphinx_doc/index.rst

```diff
@@ -0,0 +1,27 @@
+NDSL Documentation
+==================
+
+NDSL is domain-specific language for scientific computing in Python, it supports most of the Python language semantics, with performance compartible as the native C and C++ programming language.
+
+
+NDSL has been used as the backend of Pace model (https://github.com/NOAA-GFDL/pace), which runs on a laptop using Python-based backend, and on thousands of heterogeneous compute nodes of a large supercomputer using C/C++ and Cuda/HiP backend.
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+   overview
+   users_guide
+   program_guide
```

Comment:
The users guide is not a users guide it's a developer guide or a technical guide. Please move the content of it into a "NDSL developer technical documentation" and "Users guide" should be a user forward, no technical detail, but how to code with NDSL.

While you are at it introduce a "porting from Fortran" section